src/selection_heuristics.jl

```diff
@@ -39,20 +39,20 @@ with measures that are neither `:loss` nor `:score` are reset to zero.
 struct NaiveSelection <: SelectionHeuristic
     weights::Union{Nothing, Vector{Real}}
 end
-NaiveSelection(; weights=nothing) =
-    NaiveSelection(weights)
-
+function NaiveSelection(; weights=nothing)
```

Comment:
I feel you've changed the behaviour of `NaiveSelection` shouldn't the docstring be updated?

Reply:
Ok i see. The behaviour hasn't changed

Reply:
No just added a check.